docs/examples/numberinsight.md

```diff
@@ -6,9 +6,12 @@ permalink: examples/numberinsight
 * [Number Insight Basic/Standard](#number-insight-basic-standard)
 * [Number Insight Advanced](#number-insight-advanced)
 
+Use Number Insight to get additional data about a phone number and check its validity. See also:
```

Comment:
This one seems to be in a different format to the others. (You're using bullet points here, but the others you're just keeping them as links in a sentence.) Not wrong, just wondering if that was intentional.